plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -192,9 +192,36 @@ function generateAtomDocument() {
 				$pkpDetails->setAttribute('checksumValue', md5_file($packageFile));
 				break;
 		}
-
+		
 		$entry->appendChild($pkpDetails);
 		$atom->appendChild($entry);
+		
+		$license = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'license');
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'openAccessPolicy', $journal->getLocalizedSetting('openAccessPolicy')));
```

Comment:
Some settings are localized (may be different in different languages), some are not. In cases where they're localized, you're generally guaranteed to have a value in the journal's primary locale, and may optionally have a value in other locales.

When using a `getLocalizedXyz` function, it first tries the user's current locale, and if a value exists for that locale, it gets used. If not, it falls back on the value for the journal's primary locale, which should always exist.

In most import/export scenarios the user's current locale is irrelevant -- the export should represent the content the same regardless of the user's locale. So it makes sense to force the use of the journal's primary locale, which should be something like...

    $journal->getSetting('settingName', $journal->getPrimaryLocale());